docs/3.0.0-beta.x/guides/docker.md

```diff
@@ -0,0 +1,273 @@
+# Containerizing Strapi with Docker
+
+Docker greatly simplifies the deployment process of Strapi to various cloud providers and enables best practices in devops by building your code once and promoting through environments.
+
+## Prerequisites
+
+The easiest way to get started with docker is by downloading Docker Desktop and using your personal Docker Hub repositories
+
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+- [Docker Hub](https://hub.docker.com)
+
+## Building the Application
+
+Create a `.dockerignore` file in the application root to ignore some directories, similar to `.gitignore`.
+
+**Path —** `./project/.dockerignore`
+
+```
+node_modules
+build
+.cache
+```
+
+Create a `Dockerfile` in the application root.
+
+**Path —** `./project/Dockerfile`
+
+:::: tabs
+
+::: tab yarn
+
+```docker
+FROM node:alpine
+
+WORKDIR /app
+
+COPY ./package.json ./
+COPY ./yarn.lock ./
+
+RUN yarn install
+
+COPY . .
+
+RUN yarn build
+
+EXPOSE 1337
+
+CMD ["yarn","start"]
+```
+
+:::
+
+::: tab npm
+
+```docker
+FROM node:alpine
+
+WORKDIR /app
+
+COPY ./package*.json ./
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
+
+EXPOSE 1337
+
+CMD ["npm", "run", "start"]
+```
+
+:::
+
+::::
+
+Build the image and push to your Docker Hub repository with the following
+
+**Path —** `./project/`
+
+```bash
+docker build -t dockerhubaccount/project:latest .
+
+docker push dockerhubaccount/project:latest
+```
+
+You can run a single container locally with the following.
+
+```bash
+docker run -it -e NODE_ENV=development -p 1337:1337 dockerhubaccount/project
```

Comment:
Running `yarn start` with `NODE_ENV=development` won't allow users to modify the content types. You should should how to run yarn develop with the command option of docker run